src/zarr/storage/_fsspec.py

```diff
@@ -30,6 +32,45 @@
 )
 
 
+def _make_async(fs: AbstractFileSystem) -> AsyncFileSystem:
+    """Convert a sync FSSpec filesystem to an async FFSpec filesystem
+
+    If the filesystem class supports async operations, a new async instance is created
+    from the existing instance.
+
+    If the filesystem class does not support async operations, the existing instance
+    is wrapped with AsyncFileSystemWrapper.
+    """
+    import fsspec
+    from packaging.version import parse as parse_version
+
+    fsspec_version = parse_version(fsspec.__version__)
+    if fs.async_impl and fs.asynchronous:
+        # Already an async instance of an async filesystem, nothing to do
+        return fs
+    if fs.async_impl:
+        # Convert sync instance of an async fs to an async instance
+        import json
```

Comment:
can we declare this import at the top of the file?

Reply:
moved in [`3b06c53` (#2774)](https://github.com/zarr-developers/zarr-python/pull/2774/commits/3b06c536cc5398a83b81eea7fc95abcdf3eaa37f)